packages/common/components/content/blocks/page-header.marko

```diff
@@ -1,11 +1,24 @@
 import { getAsObject } from '@base-cms/object-path';
 
 $ const content = getAsObject(input, 'content');
+$ const specialTypes = ['event', 'webinar', 'whitepaper'];
 
 <endeavor-breadcrumbs-website-section section=content.primarySection />
 <cms-content-name tag="h1" class="page-wrapper__title" obj=input.content />
 <cms-content-teaser tag="p" class="page-wrapper__deck" obj=input.content />
 <if(content.type !== 'contact')>
   <endeavor-content-block-attribution content=input.content />
-  <cms-content-published obj=input.content />
+</if>
+<if(content.type !== 'contact' && !specialTypes.includes(content.type))>
```

Comment:
These `<if>` statements should probably be refactored to be `<if>` and `<else-if>` blocks.
For example:
```marko
<if content.type === "event">

</if>
<else-if content.type === "webinar">

</else-if>
<else-if content.type === "whitepaper">

</else-if>
<else-if content.type !== "contact">

</else-if>
<else>

</else>
```

This should actually eliminate the need for the `specialTypes` constant on L4

Reply:
I don't agree with this approach, as nesting positive and negative statements in that way could potentially cause readability and execution issues

I have adjusted the array name to be more explicit, and included 'contact' in the list, to make the final statement more legible as to its purpose.

Reply:
Until another condition needs to be added - or the array values of `noPublishDateContentTypes` conflict with a hardcoded `<if content.type>` condition. :)